pylintrc

```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
 # ***************************
 #
-# This file was generated by edx-lint: https://github.com/edx/edx-lint
+# This file was generated by edx-lint: https://github.com/openedx/edx-lint
```

Comment:
I think this file needs to be regenerated rather than edited directly.

Reply:
@timmc-edx yep! A few people called this out. Could you re-review now that I've run `edx_lint`?